content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -0,0 +1,240 @@
+    clientSecretFromEnv: AAD_APP_SECRET_ENV_VAR_NAME # Optional. You can use this instead of `clientSecret` parameter.
+    tenantIdFromEnv: AAD_APP_TENANT_ID_ENV_VAR_NAME # Optional. You can use this instead of `tenantId` parameter.
+```
+
+**Parameter list:**
+
+- `endpoint` - The endpoint to query your Data Explorer Cluster.
+- `databaseName` - The name of the Data Explorer Database to query.
+- `query` - Data Explorer query.
+- `threshold` - Value that is used as a threshold to calculate # of pods for scale target.
+- `tenantId` - Id of the Azure AD tenant.
+- `clientId` - Id of the Azure AD application.
+- `clientSecret` - Password of the Azure AD application.
+
+The authentication parameters could be provided using environmental variables, instead of setting them directly in metadata. Here is a list of parameters you can use to retrieve values from environment variables:
+
+- `tenantIdFromEnv` - An environmental variable name, that stores Azure AD tenant id. (Optional)
+- `clientIdFromEnv` - An environmental variable name, that stores application id of your Azure AD Application. (Optional)
+- `clientSecretFromEnv` - An environmental variable name, that stores password of the Azure AD application. (Optional)
+
+### Query Guidance
+
+It is important to design your query to return 1 row. A good practice is to add `| limit 1` at the end of your query.
+
+The only supported data types for your query result are `real`, `int` or `long`.
+
+Be careful with defining `pollingInterval` and using long-running queries. Make sure to test your query before using it.
+
+### Authentication Parameters
+
+You can use the `TriggerAuthentication` CRD to configure the authentication by providing a set of Azure Active Directory credentials or by using pod identity.
+
+The AD identity that will be used requires `DatabaseViewer` role to query metrics from the Data Explorer Cluster.
+
+💡You can use [this guide ](https://docs.microsoft.com/en-us/cli/azure/kusto/database?view=azure-cli-latest#az-kusto-database-add-principal) to assign your principal with the right access permissions through the Azure CLI.
+
+**Credential based authentication:**
+
+- `clientId` - Id of the Azure AD application. Use [this](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) guide to create your service principal.
+- `clientSecret` - Password of the Azure AD application.
+- `tenantId` - Id of the Azure AD tenant. Use [this](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-show) guide to retrieve your tenant id.
+
+
+**Pod identity based authentication:**
+
+[Azure Active Directory pod-managed identity](https://docs.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity) can be used.
+
+in place of credential based authentication. The following section contains an example of a `TriggerAuthentication` using pod identity.
+
+### Example
```

Comment:
Would you mind aligning with the approach from https://github.com/kedacore/keda-docs/pull/660 for providing multiple samples please?